tests/unit/slowlog.tcl

```diff
@@ -42,7 +51,7 @@ start_server {tags {"slowlog"} overrides {slowlog-log-slower-than 1000000}} {
         set e [lindex [r slowlog get] 0]
         assert_equal [llength $e] 6
         if {!$::external} {
-            assert_equal [lindex $e 0] 107
+            assert_equal [lindex $e 0] 106
```

Comment:
Just curious, what is this magic number?

Reply:
It's just the build up of all the unique id of the slowlog entries. I did a reset after the first entry.

Note: With the old code the entries were getting created and cleaned up in the same flow (the id(s) would have incremented a lot more without this change).

Reply:
OK